src/main/java/eu/dissco/core/datacitepublisher/security/WebSecurityConfig.java

```diff
@@ -0,0 +1,33 @@
+package eu.dissco.core.datacitepublisher.security;
+
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.http.SessionCreationPolicy;
+import org.springframework.security.web.SecurityFilterChain;
+
+@RequiredArgsConstructor
+@Configuration
+@EnableWebSecurity
+public class WebSecurityConfig  {
+
+  private final JwtAuthConverter jwtAuthConverter;
+
+  @Bean
+  public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Exception {
+    http.authorizeHttpRequests(authorizeHttpRequests -> authorizeHttpRequests
+        .anyRequest().authenticated());
```

Comment:
I feel like authenticated would be enough as anyone can authenticate. Maybe we put this behind a role, maybe the `orchestration-admin` or see if we can create a generic `admin` role

Reply:
i'll add the orchestration-admin role for now. i was thinking this endpoint would only be accessed by another machine, maybe the handle api, but we don't have a pipeline for that yet.